sbtool-sign-kernel

```diff
@@ -62,10 +62,16 @@ usage() {
 check_commands() {
     for command in "$@"; do
         if ! command -v "$command" > /dev/null; then
-            if ! $QUIET; then
-                error "$command is missing"
-            else
-                exit 1
+            for i in /usr/src/linux-obj/$(uname -m)/*/scripts/"$command" ; do
+                if [ -x "$i" ] ; then
+                    scriptdir="$(dirname "$i")"
+                    quiet_message "Using $command from $scriptdir"
+                    PATH="$PATH:$scriptdir"
+                    break
```

Comment:
This could be 'continue 2' and then the duplicate 'command -v' could be removed.

Reply:
changed to continue 2  and added riscv64